From d0601d4a2c63de2d276119859620b3c095956858 Mon Sep 17 00:00:00 2001
From: Erez_Zadok <ezk@cs.sunysb.edu>
Date: Wed, 6 Jun 2007 01:20:15 -0400
Subject: [PATCH] bugfix: add MODULE_LICENSE to mtdsuper module so it loads

This is a temp fix in rc4.  Also reported this bug to MTD maintainer.
---
 drivers/mtd/mtdsuper.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/mtdsuper.c b/drivers/mtd/mtdsuper.c
index 9b430f2..75450a3 100644
--- a/drivers/mtd/mtdsuper.c
+++ b/drivers/mtd/mtdsuper.c
@@ -232,3 +232,5 @@ void kill_mtd_super(struct super_block *sb)
 }
 
 EXPORT_SYMBOL_GPL(kill_mtd_super);
+
+MODULE_LICENSE("GPL");
-- 
1.5.4.3

